src/elm/Page/Profile.elm

```diff
@@ -231,7 +231,9 @@ view_ loggedIn profile model =
                                 , ( "circle-background", True )
                                 , ( "circle-background--primary", True )
                                 ]
-                            , onClick DownloadPdf
+
+                            -- TODO: Should be passphrase, not privateKey
```

Comment:
The thing is that the user while downloading PDF from the Profile may get the private key instead of 12 words if the page was reloaded.

This TODO is actually related to the #211 issue (registration and login work fine). I think it's better to handle it there.